src/compiler/visitorPublic.ts

```diff
@@ -580,20 +581,20 @@ export function visitCommaListElements(elements: NodeArray<Expression>, visitor:
  * @param visitor The callback used to visit each child.
  * @param context A lexical environment context for the visitor.
  */
-export function visitEachChild<T extends Node>(node: T, visitor: Visitor, context: TransformationContext): T;
+export function visitEachChild<T extends Node>(node: T, visitor: Visitor, context: TransformationContext | undefined): T;

```

Comment:
Also, debating whether or not `visitLexicalEnvironment`, `visitParameterList`, `visitFunctionBody`, and `visitIterationBody` need this change. Those are pretty explicitly about working with the context and within a transform, though, so I'm leaning towards no.

https://github.com/search?q=%2Fts%5C.%28visitLexicalEnvironment%7CvisitParameterList%7CvisitFunctionBody%7CvisitIterationBody%29%2F+%28language%3ATypeScript+OR+language%3AJavaScript%29&type=code